src/recording/Transcoder.ts

```
@@ -219,25 +219,29 @@ export class Transcoder extends EventEmitter {
         }
     }
 
-    // Simplified chunk processing
+    // Simplified chunk processing - VIDEO PROCESSING DISABLED
     public async uploadChunk(
         chunk: Buffer,
         isFinal: boolean = false,
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)